### PR TITLE
Add test for consensus leasing validation to introspector

### PR DESCRIPTION
**Pull request by xiancao**
Add test for consensus leasing validation to introspector.

Jenkins result:
https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/11581/
https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/11580/

(pani) I run the all the tests in integration test profiles 
 https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/11605/ 
All tests passed including new test class ItConsensusLeasingValidation

### CODE REVIEW

**Review by alai8 — APPROVED** [review]

**Review by anpanigr — COMMENTED** [review]
I do not see the test class (ItConsensusLeasingValidation) result in 
https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/11581/testReport/oracle.weblogic.kubernetes/

**Comment by xiancao** [comment]
> I do not see the test class (ItConsensusLeasingValidation) result in
> https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/11581/testReport/oracle.weblogic.kubernetes/

@anpanigr This is because the above is a sequential run and the test takes long than 800 minutes and got timed out. Here is the individual run for ItConsensusLeasingValidation:
https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/11577/

**Comment by anpanigr** [comment]
> > I do not see the test class (ItConsensusLeasingValidation) result in
> > https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/11581/testReport/oracle.weblogic.kubernetes/
> 
> @anpanigr This is because the above is a sequential run and the test takes long than 800 minutes and got timed out. Here is the individual run for ItConsensusLeasingValidation: https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/11577/

Just I want to assert that it does not break any suite in parallel run.  Also please add the corresponding tag

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItConsensusLeasingValidation.java [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+import static oracle.weblogic.kubernetes.utils.PodUtils.verifyIntrospectorPodLogContainsExpectedErrorMsg;
+import static oracle.weblogic.kubernetes.utils.SecretUtils.createSecretWithUsernamePassword;
+import static oracle.weblogic.kubernetes.utils.ThreadSafeLogger.getLogger;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+// Test introspector will fail with error msg if the consensus leasing is configured in the cluster or the database
+// leasing is configured but there is no datasource.
+@DisplayName("Test introspector will fail with error msg if the consensus leasing is configured in the cluster")
+@IntegrationTest
```
[the remark this answers is not in the document]
Comment (xiancao) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItConsensusLeasingValidation.java:38]: added the tags

**Review by anpanigr — APPROVED** [review]

**Pull request merged**